.gitignore

```diff
@@ -1,4 +1,5 @@
 .HOLMK
 .hollogs
 .holobjs
+HOL
```

Comment:
is this intentional? I think it's probably better not to recommend to build HOL _within_ the verifereum working directory, but rather externally (e.g. alongside it)

Reply:
Gotcha. Wasn't sure how people with multiple per-project HOLs tend to do it.